db/migrate/20160415200213_add_missing_indexes.rb

```diff
@@ -1,12 +1,11 @@
-class AddMissingIndexes < ActiveRecord::Migration
+class AddMissingIndexes < ActiveRecord::Migration[5.2]
```

Comment:
If these migrations were generated in Rails 4, should we have `[5.2]` here or should it be `[4.2]`? Had a look and that's what we did in WEX.

Reply:
Done!